src/shared_gui_components/WorkflowController.cpp

```diff
@@ -501,6 +501,8 @@ void MeasureStepItem::setArgument(const measure::OSArgument& argument) {
       m_step.setArgument(argument.name(), argument.valueAsDouble());
     } else if (argument.type() == measure::OSArgumentType::Integer) {
       m_step.setArgument(argument.name(), argument.valueAsInteger());
+    } else if (argument.type() == measure::OSArgumentType::Path) {
+      m_step.setArgument(argument.name(), openstudio::toString(argument.valueAsPath()));
```

Comment:
Without this you, you get valueAsString, and it gets quoted in the workflowJSON as 

```
    "my_path_argument": "\"/the/path/\""
```

and when you run, Ruby for eg can't understand it.